filament/backend/include/backend/DriverEnums.h

```diff
@@ -49,7 +49,7 @@ static constexpr uint64_t SWAP_CHAIN_CONFIG_ENABLE_XCB = 0x4;
 static constexpr uint64_t SWAP_CHAIN_CONFIG_APPLE_CVPIXELBUFFER = 0x8;
 
 static constexpr size_t MAX_VERTEX_ATTRIBUTE_COUNT = 16; // This is guaranteed by OpenGL ES.
-static constexpr size_t MAX_SAMPLER_COUNT = 17;          // Does not match the Adreno Vulkan driver but does satisfy authoring shader sampler needs
+static constexpr size_t MAX_SAMPLER_COUNT = 16;          // Does not match the Adreno Vulkan driver but does satisfy authoring shader sampler needs
```

Comment:
This should have been done via git revert, not a manual fix. Sorry I missed that when I reviewed it. The comment is wrong now